rtc-video-quality/generate_data.py

```diff
@@ -307,7 +313,7 @@ def generate_metrics(results_dict, job, temp_dir, encoded_file):
         elif metric == 'Nframes':
             layer_frames = int(value)
             results_dict['frame-count'] = layer_frames
-
+    results_dict["psnr-dmos"] = psnr_to_dmos(results_dict['avg-psnr'])
```

Comment:
https://google.github.io/styleguide/pyguide.html#310-strings says:
>Be consistent with your choice of string quote character within a file. Pick ' or " and stick with it. 

Since you're modifying existing code, respect the local style (which appears to be single quotes).